### DISCUSSION TITLE
How to configure rustdesk to disable the first direct connection?

### DISCUSSION THREAD

**Discussion by rhk-hu** [rustdesk/rustdesk#8882] [Q&A] [answered]
We have a self hostet Rustdeskserver, but the first rustdesk clinet connection to a remote rustdesk one succeds directly, and fails, because Firewall blocks the varous tcp-outgoing Ports from range ~50000-60000, is wating a timeout  and only after 20 seconds the clients speaks with rustdeskserver and connect.
Subsequent connection ones are gone smooth. Howto configure rustdesk to disable the first direct connection?

**Comment by danielmarschall** [comment]
Does it help when you append "/r" to the ID on the controlling side? This will force the connection to be sent over the relay server.

**Comment by rhk-hu** [comment]
Oh yes with /r is working immediately. Is there a possibility to configure
rustdesk to take always the relay server?

many thanks for this trick

**Reply by rustdesk** [reply, marked as answer]
Pro

![image](https://github.com/user-attachments/assets/6cea0798-eaea-4701-adc2-528697069c49)
